packages/paste-core/components/side-modal/src/SideModalFooterActions.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+import type {BoxElementProps, BoxStyleProps} from '@twilio-paste/box';
+
+export interface SideModalFooterActionsProps extends React.HTMLAttributes<HTMLDivElement> {
+  children: NonNullable<React.ReactNode>;
+  element?: BoxElementProps['element'];
+  justify?: BoxStyleProps['justifyContent'];
```

Comment:
question: of this full list of all flex justify options, do you have opinions on whether some of these _shouldn't_ be used because they might "look bad"? Should we restrict to a collection of options that we as a design team think encourages good design?

Reply:
This came up from a talk with @TheSisb. The way modal does it, theres only 2 options 'start' and 'end' which map to 'flex-start' and 'flex-end'. This felt weird because it is renaming the options so people dont actually know what theyre getting... Evergreen also has a use case for 'space-between' where there is a back button on the left and the save/cancel button on the right.

I'd be fine reducing the options to just flex-start/end and space-between.

Reply:
Would it be weird to also rename the prop justifyContent? That would be a change from how modal does it but I think its a better practice.